contrib/builder/rpm/opensuse-12.3/Dockerfile

```diff
@@ -0,0 +1,16 @@
+#
+# THIS FILE IS AUTOGENERATED; SEE "contrib/builder/rpm/generate.sh"!
+#
+
+FROM jess/opensuse:12.3
```

Comment:
This is just a POC we will make it official after I'm done w testing if the
official lib gods are in my favor, I have a sweet automated setup

On Friday, January 15, 2016, Sebastiaan van Stijn <notifications@github.com>
wrote:

> In contrib/builder/rpm/opensuse-12.3/Dockerfile
> <https://github.com/docker/docker/pull/19378#discussion_r49926331>:
>
> > @@ -0,0 +1,16 @@
> > +#
> > +# THIS FILE IS AUTOGENERATED; SEE "contrib/builder/rpm/generate.sh"!
> > +#
> > +
> > +FROM jess/opensuse:12.3
>
> do we want this to be in the docker/ organization?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/19378/files#r49926331>.
>